sample-workspace/neo-servers.json

```diff
@@ -1,14 +1,12 @@
-[
-    {
-        "name": "Neo Main Net",
-        "url": "http://seed2.ngd.network:10332"
-    },
-    {
-        "name": "Neo Test Net",
-        "url": "https://test3.cityofzion.io"
-    },
-    {
-        "name": "Neo Test Net - getunspents supported",
-        "url": "http://seed5.ngd.network:20332"
+{
+    "neo-rpc-uris": [
+        "http://seed2.ngd.network:10332",
+        "https://test3.cityofzion.io",
```

Comment:
test3.cityofzio.io. shows up twice now. SHould we hide one?

Reply:
Done. I added some others that are not returned by the API instead.